### DISCUSSION TITLE
Text, Image and Video Content Management with different format, size, dimension and version

### DISCUSSION THREAD

**Discussion by firozemzr** [OSBDAlliance/headlesscms#5] [Ideas]
Headless CMS should allow 

1. Addition
2. Deletion (Archive, Mark Delete and Total Purge)
3. Edit (with version tracking)
4. Retrieved

of text, image and video contents using api.

Detail meta data of content should be managed. The content can be retrieved using different combination of metadata including but not limited to contentid, category, subcategory, keyword, tag, title etc.

Specific format, size, dimension or version of content can be retrieved

**Comment by mohammad-arif-rahim** [comment]
Not clear what we are trying to achieve here. Are we planning to provide an editor? then it is not headless cms.

**Reply by firozemzr** [reply]
No, no editor, a CMS needs to deliver contents of specific size/dimension, length etc of a content based on request/user agent..

Its a very basic but must have requirement for CMS as the destination devices comes in many different screen size and capabilites.. Also now even different social media requires different length, size/ratio of contents.

CMS has to cater to variations of a single content and must be able to deliver them as per called/inout parameters.
